src/components/Profile.js

```diff
@@ -88,17 +139,45 @@ class Profile extends Component {
               </div>
 
               <h4 className="mt-5 mb-2">Deine Rezepte</h4>
-              <p className="px-0 mx-0">
-                <Link to="/add-recipe" className="nav-link addButton">
-                  <FiPlusCircle className="addButton" /> Neues hinzufügen
-                </Link>
-              </p>
+              {this.state.user.userId === this.props.registerUser ? (
+                <p className="px-0 mx-0">
+                  <Link to="/add-recipe" className="nav-link addButton">
+                    <FiPlusCircle className="addButton" /> Neues hinzufügen
+                  </Link>
+                </p>
+              ) : null}
 
               <div className="row">
-                <RecipePreview />
-                <RecipePreview />
-                <RecipePreview />
-                <RecipePreview />
+                {!this.state.loading && this.state.recipes.length > 0 ? (
+                  this.state.recipes.map((recipe, index) => (
+                    <div className="col-12 col-lg-6 mr-auto">
```

Comment:
Hier müsste noch der `key` auf den `div` und dann den auf `RecipeCard` löschen. Dann bekommt man kein Error mehr in der Konsole.